### PR TITLE
New fine-grained destination limit across reporting origins - explainer update

### PR DESCRIPTION
**Pull request by akashnadan**
Addresses #771

### CODE REVIEW

File: EVENT.md [hunk 1]
```diff
@@ -943,6 +943,16 @@ any one reporting origin.
 Strawman: 100 distinct destination sites per-{source site, reporting origin},
 applied to all unexpired sources regardless of type at source time.
 
+#### Limiting the number of unique destinations per source site
+
+To further reduce the possibility of a history reconstruction attack, the browser can also limit the number of `destination` eTLD+1s represented by source-sites.
+
+Strawman: 200 distinct destination sites per-{source site, 1 minute}
```
Comment (csharrison) [inline EVENT.md:950]: Please update the "denial of service section" to add this possible denial of service attack.

File: EVENT.md [hunk 2]
```diff
@@ -943,6 +943,16 @@ any one reporting origin.
 Strawman: 100 distinct destination sites per-{source site, reporting origin},
 applied to all unexpired sources regardless of type at source time.
 
+#### Limiting the number of unique destinations per source site
+
+To further reduce the possibility of a history reconstruction attack, the browser can also limit the number of `destination` eTLD+1s represented by source-sites.
```
Comment (apasel422) [inline EVENT.md:948]: What does "represented" mean in this case?

Comment (akashnadan) [reply EVENT.md]: It means the destination sites that were specified by the source registrations. Changing this to "registered" to make it more clear

File: EVENT.md (hunk 1 above)
Comment (akashnadan) [reply EVENT.md:950]: i have added a call out in that section

**Pull request merged**